### ISSUE TITLE
Fix false positives for "/" in function-whitespace-after

### ISSUE THREAD

**Issue by dsine-de** [stylelint/stylelint#3132]
Allow "/" as an allowed character in the "function-whitespace-after" rule to allow e.g. line-height shorthand value in "font" property.

> Which rule, if any, is this issue related to?

`function-whitespace-after`

> What CSS is needed to reproduce this issue?

```css
.foo {
  font: calc(16px + .2vw)/1 sans-serif;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

`8.4.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

gulp-stylelint

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No.

> What did you expect to happen?

Allowance of the "/" character as default.

> What actually happened (e.g. what warnings or errors you are getting)?

Expected whitespace after ")" (function-whitespace-after)

**Comment by dsine-de** [comment]
This probably conflicts with checking for division character between two functions...

**Comment by ntwb** [comment]
Via  @Arcanemagus in the PR https://github.com/stylelint/stylelint/pull/3131#issuecomment-359127260

> A concern I had was that this may be valid for `font`, but this PR is making it valid for _any_ position after a `)`. Does that actually make sense? Or should this be limited to `font` somehow?

**Comment by ntwb** [comment]
I probably should have researched this issue a little more before commenting/reviewing  the PR 😉 

I've dug into the past history of `ACCEPTABLE_AFTER_CLOSING_PAREN` in the `function-whitespace-after` rule and I think it will be fine to not have to restrict it to the `font` rule.

In https://github.com/stylelint/stylelint/pull/1391 was when `ACCEPTABLE_AFTER_CLOSING_PAREN` the colon `:` was also added, it's currently undocumented in the [readme](https://github.com/stylelint/stylelint/blob/master/lib/rules/function-whitespace-after/README.md) or rules [inline docs](https://github.com/stylelint/stylelint/blob/master/lib/rules/function-whitespace-after/index.js#L46-L47) and following the logic in #1391 I don't see the addition of the `/` being of a concern as it will only be limited to the `function-whitespace-after` rule and is not used in any stylelint utils.

**Comment by ntwb** [comment]
> This probably conflicts with checking for division character between two functions...

With what I said above, let's also add some tests in the `function-whitespace-after` to determine if indeed that is the case
